src/store.rs

```diff
@@ -19,8 +30,107 @@ use crate::{
 };
 
 #[derive(Deref)]
+/// A wrapper around the internal rusqlite Connection
```

Comment:
```suggestion
/// A Nix database connection.
```

Also move all doc comments above #[attributes].